msteams-platform/sbs-botbuilder-linkunfurling.yml

```diff
@@ -0,0 +1,474 @@
+          
+    1. In the panel that appears, enter `access_as_user` as the **Scope name**.
+  
+    1. Set **Who can consent?** to `Admins and users`.
+  
+    1. To configure the admin and user consent prompts with appropriate values for `access_as_user` scope, provide the following information in the fields:</br>
+    
+         * Enter `Teams can access the user’s profile` as **Admin consent display name**.
+
+         * Enter `Allows Teams to call the app’s web APIs as the current user` as **Admin consent description**.
+
+         * Enter `Teams can access the user profile and make requests on the user’s behalf` as **User consent display name**.
+
+         * Enter `Enable Teams to call this app’s APIs with the same rights as the user` as **User consent description**.
+  
+    1. Ensure that **State** is set to **Enabled**.
+  
+    1. Select **Add a scope** to save.
+
+        :::image type="content" source="../msteams-platform/assets/images/Teams-link-unfurling/addscope.png" alt-text="Add a scope" border="true"::: 
+        
+        > [!NOTE]
+        > The **Scope name** should match with the **Application ID** URI with `/access_as_user` appended at the end.</br>
+           `api://your ngrok/botid-00000000-0000-0000-0000-000000000000/access_as_user`
+
+          :::image type="content" source="../msteams-platform/assets/images/Teams-link-unfurling/scope.png" alt-text="Scopes" border="true"::: 
+        
+    1. In the **Authorized client applications** section, identify the applications that you want to authorize for your app’s web application. 
+    
+    1. Select **Add a client application**. 
+
+       :::image type="content" source="../msteams-platform/assets/images/meetings-side-panel/selectclientapp.png" alt-text="Select client application" border="true":::
+       
+    1. Enter **Client ID**: `1fec8e78-bce4-4aaf-ab1b-5451cc387264` for Teams desktop application. 
+   
+       :::image type="content" source="../msteams-platform/assets/images/meetings-side-panel/addclientapplication1.png" alt-text="Add client application" border="true":::
+       
+       Enter **Client ID**: `5e3ce6c0-2b1f-4285-8d4b-75ee78787346` for Teams web application.
+
+       :::image type="content" source="../msteams-platform/assets/images/meetings-side-panel/addclientapplication2.png" alt-text="Add client application" border="true":::
+      
+    1. Select **Authorized scopes**.
+
+       :::image type="content" source="../msteams-platform/assets/images/Teams-link-unfurling/authorizedscope.png" alt-text="Add client application" border="true":::
+      
+       The following image displays the **Client Id**:
+
+       :::image type="content" source="../msteams-platform/assets/images/meetings-side-panel/clientapps.png" alt-text="Client app" border="true"::: 
+         
+    1. In the left panel, select **API Permissions**. 
+
+       > [!NOTE]
+       > Users need to consent to the API permissions only if the AAD app is registered in a different tenant.
```

Comment:
FYI, @surbhigupta12 . We approved an instance of a branding issue here. We shouldn't.
It is never AAD. This instance is fixed via PR #6759